doc/Options/zCOBOL_filetypes.md

```diff
@@ -0,0 +1,12 @@
+# zCOBOL file types
+
+TYPE | Format | File Description       | File or Report Format Description
+-----|--------|------------------------|---
+CBL  | ASCII  | COBOL source program   | 1-6 sequence #, 7 Comment if not space, 8-11 area A, 12-72 area B
+CPZ  | ASCII  | COBOL copy book member | 1-6 sequence #, 7 Comment if not space, 8-11 area A, 12-72 area B
+MLC  | ASCII  | Macro assembler source program generated by phase 1 of the zcobol compiler which uses zcobol.class regular expression based parser in z390.jar to read CBL source file and create MLC source file in one pass. | Macro call. for each COBOL statement starting in area A and for each COBOL verb found in area B.  Working storage data items are mapped to WS macro call with level as first parameter.  Each macro call name is followed by positional  parameters found following verb up to next verb or period.   Periods are mapped to PERIOD macro call.  Parameters of the form keyword(..) are passed as single parameter.  Other ( and ) are passed as separate parameter in quotes.
+BAL  | ASCII  | HLASM compatible source code generated by phase 2 of the zCOBOL compiler when using ZC390C or ZC390CLG commands.|HLASM compatible source statements generated by the zcobol+zcobol\z390 macros during expansion of the generated MLC file.
```

Comment:
Line 8. 
1. I believe that ZC390Cx has been replaced by CBLCx, and there is a CBLCL. Therefore,
"... using ZC390C or ZC390CLG commands." -> "... using CBLC, CBLCL, or CBLCLG commands.".
2. In the 4th column. "... the zcobol+zcobol\z390 macros ..." -> "... the zcobol\mac macros ...".

You should run these two by Don to be sure.

Reply:
@dsh33782 -- Did you have any comments on the above question from John?

Reply:
Fixed aff28bd